packages/config/config/devices/0x019b/zm_single_relay_16.json

```
@@ -0,0 +1,228 @@
+{
+	"manufacturer": "HeatIt",
+	"manufacturerId": "0x019b",
+	"label": "ZM SINGLE RELAY 16",
+	"description": "Heatit ZM Single Relay 16A",
+	"devices": [
+		{
+			"productType": "0x0004",
+			"productId": "0x3500",
+			"zwaveAllianceId": 4062
+		}
+	],
+	"firmwareVersion": {
+		"min": "0.0",
+		"max": "255.255"
+	},
+	"associations": {
+		"1": {
+			"label": "Lifeline",
+			"maxNodes": 5,
+			"isLifeline": true
+		},
+		"2": {
+			"label": "External Relay Control S1",
+			"maxNodes": 5
+		},
+		"3": {
+			"label": "Control External Start/Stop S2",
+			"maxNodes": 5
+		},
+		"4": {
+			"label": "External Relay Control S2",
+			"maxNodes": 5
+		},
+		"5": {
+			"label": "Control External Start/Stop S2",
+			"maxNodes": 5
+		}
+	},
+	"paramInformation": {
+		"1": {
+			"label": "Load Limit",
+			"description": "1-16 A",
+			"valueSize": 1,
+			"minValue": 1,
+			"maxValue": 16,
+			"defaultValue": 16,
+			"unsigned": true
+		},
+		"2": {
+			"label": "Power Shutdown Actions",
+			"description": "0: Disabled, 1-32767: minute(s) delay before attempting to turn back on",
```

I think this could be better worded as 
```suggestion
			"label": "Turn On Delay After Emergency Off",
```
or something.

The unit should be in its separate field.